README.md

```diff
@@ -652,14 +652,14 @@ API reference for CUBA React components can be found [here](http://cuba-platform
 
 <a name="react-client-routing"/>
 
-### Routing
+### Routing And Menu
```

Comment:
We have a requirement to follow [these rules](https://www.grammarcheck.net/capitalization-in-titles-101/) of capitalization in titles. According to them, `and` should not be capitalized (prepositions and conjunctions of four or fewer letters should not be capitalized).